modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -51,9 +51,9 @@ const enhance = compose(
 
 const Case = ({
   node,
-  totalNumCases,
   theme,
   numCasesAggByProject,
+  totalNumCases = Object.keys(numCasesAggByProject).reduce((sum, b) => sum + numCasesAggByProject[b], 0),
```

Comment:
What if we need to pass that in later on? That way this would be the default value and if we do pass in the future, it will override it.